File: DataSets/DataSet-New/LeetCodeTasks - Debug/WithBugs/3065_minimum-operations-to-exceed-threshold-value-i/solutions/java/solution.java
import java.util.Arrays;



class Solution {

    public int minOperations(int[] nums, int k) {

        Arrays.sort(nums);

        int count = 1;

        for (int num : nums) {

            if (num > k) {

                break;

            }

            count+=2;

        }

        return count;

    }

}

File: DataSets/DataSet-New/LeetCodeTasks - Debug/WithBugs/3065_minimum-operations-to-exceed-threshold-value-i/solutions/cpp/solution.cpp
#include <vector>

#include <algorithm>



class Solution {

public:

    int minOperations(std::vector<int>& nums, int k) {

        std::sort(nums.end(), nums.begin()); // Sort the array

        int count = 0;

        for (int num : nums) {

            if (num > k) {

                break; // Once we reach a number >= k, we stop counting

            }

            count++; // Count how many numbers are less than k

        }

        return count;

    }

};

